### PR TITLE
Reorganize the structure of jwt generation in perf tests

### PR DESCRIPTION
**Pull request by MushuEE**
This feels like it follows SRP better and groups like things
JWTIssue and BuildJWTConfig

/cc @chaodaiG 
Fixes #

## Proposed Changes

*Move config mock to utils
*Clean up responsibility of testSetup

**Release Note**

```release-note

```

### CODE REVIEW

**Comment by chaodaiG** [comment]
lgtm, cc @argetlam-cs

File: internal/utils/utils.go [hunk 1]
```diff
@@ -117,3 +120,37 @@ func IssueJWT(t *testing.T, cfg JWTConfig) (jwtText, hmacKey string) {
 	}
 	return
 }
+
+// BuildJWTConfig creates a stub for a generic JWT given an existing database and
+// exposure keys
+func BuildJWTConfig(tb testing.TB, db *database.DB, keys []verifyapi.ExposureKey) *JWTConfig {
```
Comment (argetlam-cs) [inline internal/utils/utils.go:126]: update publish_test to use this

Comment (MushuEE) [reply internal/utils/utils.go:126]: Still working on getting this working with `iss` conflicts. Got side tracked with a high prio fix 👍

**Comment by MushuEE** [comment]
/hold 
Trying to integrate change with publish_test
*That or I can update as part of different issue

File: internal/utils/utils.go (hunk 1 above)
Comment (MushuEE) [reply internal/utils/utils.go:126]: Fixed iss, was init db in two places. Now hitting 401

**Comment by MushuEE** [comment]
/hold cancle

@chaodaiG @argetlam-cs Thanks for looking at this, the interactions of creating many HA/DBs within the publish tests made my original plan incompatible. This PR turned into more of a clean up effort than restructure of the HA init stubs.

**Pull request merged**